app/controllers/api/v1/npq_applications_controller.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require "csv"
+
+module Api
+  module V1
+    class NPQApplicationsController < Api::ApiController
+      include ApiTokenAuthenticatable
+      include ApiPagination
+
+      def index
+        respond_to do |format|
+          format.json do
+            render json: NPQApplicationSerializer.new(paginate(scope)).serializable_hash
+          end
+
+          format.csv do
+            render body: NPQApplicationCsvSerializer.new(scope).call
+          end
+        end
+      end
+
+    private
+
+      def npq_lead_provider
+        current_api_token.cpd_lead_provider.npq_lead_provider
+      end
+
+      def scope
+        s = npq_lead_provider.npq_profiles.includes(:user, :npq_course)
```

Comment:
npq_lead_provider can be nil here in theory, right?

Reply:
i've made a small change so this is no longer possibly based on the `access_scope`